stacks-network/gaia
# Feature: Update all the package versions (Besides CJS to ESM)

## Goal of PR

Updating all the packages that are used in the hub, reader and admin. 

## Implementation

Updating packages and making the appropriate changes in the code.

## Manual Testing

Set up a gaia hub on:
- GCP
- Azure
- AWS
and see that it works the same as before

## Automated Testing

Unit tests

## Submission Checklist


      contains correct TypeFlow annotations.


      would be expected to interact with them.
